packages/react/src/themes/teams/components/Icon/svg/icons/urgent.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" className={classes.svg} viewBox="8 8 16 16">
```

Comment:
don't we have outline path for these icons?

Reply:
@mnajdova I couldn't find the outline paths in ProcessedIcons. @codepretty, do we need to add them?

Reply:
Talked offline with @codepretty, I updated the PR with the outline path for Urgent, and she has requested outline path for input-invalid which should be ready in the next couple days.

Reply:
Excluding input-invalid until we get the outline path